sql/database.sql

```diff
@@ -1611,6 +1612,7 @@ CREATE TABLE `facility` (
   `oid` VARCHAR(255) NOT NULL DEFAULT '' COMMENT 'HIEs CCDA and FHIR an OID is required/wanted',
   `iban` varchar(50) default NULL,
   `info` TEXT,
+  UNIQUE KEY `uuid` (`uuid`),
   PRIMARY KEY  (`id`)
```

Comment:
@yashrajbothra , A common issue that happens when adding a new column to the facility table is that it breaks the install (we have all made this mistake; some of use including myself even more than once :) ). Just need to remember to place NULL for the `uuid` below this where the facility insert is (line 1623).